packages/contracts/package.json

```diff
@@ -3,6 +3,7 @@
     "private": true,
     "scripts": {
         "start": "hardhat node",
+        "copy:abi": "ncp  artifacts/contracts/Semaphore.sol/Semaphore.json ../utils/src/semaphore-abi.json",
```

Comment:
This script here should be enough. No need to have one in the main `package.json` file. Just make sure to add a `&& yarn copy:abi` after `hardhat compile` here.